src/components/WebsiteFooter/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { cn } from '@/lib/utils'
+import { VariantProps, cva } from 'class-variance-authority'
+import React from 'react'
+
+const websiteFooterVariants = cva(
+  ['flex', 'flex-col', 'w-full', 'items-center', 'px-16', 'py-4'],
+  {
+    variants: {
+      size: {
+        standard: [],
+        sm: [
+          '[&>div]:flex-col',
+          '[&>div]:items-center',
+          '[&>div]:gap-4',
+          '[&>div]:[&>div]:items-center'
+        ]
+      }
+    },
+    defaultVariants: { size: 'standard' }
+  }
+)
+
+const footerStatementVariants = cva(
+  ['flex', 'flex-col', 'gap-1', 'items-center', 'text-foreground-subtle'],
+  {
+    variants: {
+      size: {
+        standard: ['py-8'],
+        sm: ['py-4', 'text-sm']
+      }
+    },
+    defaultVariants: { size: 'standard' }
+  }
+)
+
+const linkRowVariants = cva(['flex', 'w-full', 'justify-between'])
+
+const linkColumnVariants = cva(
+  ['flex', 'flex-col', 'px-8', 'dark:text-white'],
+  {
+    variants: {
+      size: {
+        lg: [
+          'gap-3',
+          '[&>div]:font-semibold',
+          '[&>div]:mb-2',
+          '[&>div]:text-2xl',
+          '[&>*]:font-normal',
+          '[&>*]:text-md'
+        ],
+        md: [
+          'gap-2',
+          '[&>div]:font-semibold',
+          '[&>div]:mb-2',
+          '[&>div]:text-xl',
+          '[&>*]:font-normal',
+          '[&>*]:text-sm'
+        ],
+        sm: [
+          'items-center',
+          'gap-2',
+          '[&>div]:font-semibold',
+          '[&>div]:mb-2',
+          '[&>div]:text-xl',
+          '[&>*]:font-normal',
+          '[&>*]:text-sm'
```

Comment:
Seeing a lot of similarities here. Rather than specify these values twice, make them the default and then overwrite them in 3 if its different.